src/styles/blocks/watch.scss

```
@@ -0,0 +1,59 @@
+.watch {
+  position: relative;
+
+  width: $watch-height;
+  height: $watch-height;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    z-index: 1;
+  
+    width: 6%;
+    height: 6%;
+    
+    background-color: cadetblue;
+    border-radius: 50%;
+  
+    transform: translate(-50%, -50%);
+  }
+  
+  &__hand {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+  
+    width: 3%;
+    height: 50%;
+  
+    border-radius: 10px;
+  
+    transform: rotate(0deg) translate(-50%, -100%);
```

```suggestion
    transform: rotate(0deg)
```
Instead of using this translate you could use smth like `transform-origin: bottom-center`